tests/acceptance/SubscriberCountShortcodeCest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+use MailPoet\Test\DataFactories\Subscriber;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../DataFactories/Subscriber.php';
+
+class SubscriberCountShortcodeCest {
+  function createSubscriberCountPageWithShortcode(\AcceptanceTester $I) {
+    $I->wantTo('Create page with MP subscriber shortcode');
+    $segment_factory = new Segment();
+    $segment = $segment_factory->withName('SubscriberCount')->create();
+    $subscriber_factory = new Subscriber();
+    $subscriber_factory->withSegments([$segment])->create();
+    $pageTitle='OurSubscribers';
+    $pageContent='[mailpoet_subscribers_count\ segments="' . $segment->id . '"]';
+    $I->cli('post create --allow-root --post_type=page --post_title=' . $pageTitle . '  --post_content=' . $pageContent);
+    $I->login();
+    $I->amOnPage('/wp-admin/edit.php?post_type=page');
+    $I->waitForText($pageTitle);
+    $I->click($pageTitle);
+    $I->click('Publish');
+    //see live page with shortcode output
+    $I->click('View page');
```

Comment:
The test failed for me locally. This link was not found. Do we need to wait for it? Or is it because in Gutenberg (which is what is running on my machine) you need to click 'Publish' twice?

Reply:
It's b/ c of Gute, I thought of that when I woke up this morning. It takes two Publishes. 
On the upside, it only fails on Publish, and not elsewhere. I can fix.

Reply:
bypassed by using CLI to set status to published, which works in WP versions before and after 5.0.